src/mappings/DAOToken/mapping.ts

```diff
@@ -51,6 +52,10 @@ export function handleTransfer(event: Transfer): void {
   ent.value = event.params.value;
 
   store.set('TokenTransfer', ent.id, ent);
+
+  if (event.params.from !== event.transaction.from) {
+    updateAllowance(event.address, event.params.from, event.transaction.from);
```

Comment:
why it is under the Transfer event

Reply:
Because a transfer may reduce allowance if it was a transfer from